polymer.json

```diff
@@ -0,0 +1,19 @@
+{
+  "entrypoint": "app/1.0/index.html",
+  "shell": "app/elements/pw-shell.html",
+  "fragments": [
+    "app/elements/pw-shell.html",
```

Comment:
I think so, because we want it to get bundled, right? I'm basically only using what the `elements` folder contains, and nothing else in the `app` output.